packages/grafana-ui/package.json

```diff
@@ -31,6 +31,7 @@
     "@grafana/e2e-selectors": "7.5.0-pre.0",
     "@grafana/slate-react": "0.22.9-grafana",
     "@grafana/tsconfig": "^1.0.0-rc1",
+    "@grafana/aws-sdk": "0.0.3",
```

Comment:
@sunker we need to remove this, we cannot circular dependency between @grafana/aws-sdk and @grafana/ui  

Can we not move SigV4AuthSettings.tsx  to @grafana/aws-sdk  ? 

@mckn

Reply:
I agree this is not ideal. However, grafana/aws-sdk specifies grafana/ui as an external dependency in the rollup config, so user's won't get two copies of grafana ui. 

I think the right path forward here would be to move the aws-sdk dependency to the plugins that use SIGV4, ie prometheus and elastic. Then the connection config component from the aws-sdk could be passed to the DataSourceHttpSettings component as children or render props so that grafana ui doesn't have to depend on the aws-sdk. This will include some refactoring of the DataSourceHttpSettings component. cc @grafana/observability-squad